hack/source-manifests.sh

```diff
@@ -58,15 +58,15 @@ function dump_rook_csv() {
 
 	crd_list=$(mktemp)
 	echo "Dumping rook csv using command: $IMAGE_RUN_CMD --entrypoint=cat $ROOK_IMAGE $rook_template_dir/$rook_csv_template"
-	$IMAGE_RUN_CMD --entrypoint=cat "$ROOK_IMAGE" $rook_template_dir/$rook_csv_template > $ROOK_CSV
+	$IMAGE_RUN_CMD --platform="${GOOS}"/"${GOARCH}" --entrypoint=cat "$ROOK_IMAGE" $rook_template_dir/$rook_csv_template > $ROOK_CSV
```

Comment:
If rook is pushing image for both arches then we can skip this change for the rook image